docs/aks/aks.md

```diff
@@ -0,0 +1,13 @@
+# Running DICOM Server in AKS
```

Comment:
This readme seems to try and combine 2 things.
1. Preparing a cluster for using dicom helm chart
    1. I think a logical place for this information is in a readme in dicom helm chart 
2. Setting up ADO pipeline for CI.
    1. I think a logical place for this information is in docs/development/AzureDevopsPipelines.md (this file doesn't exist)
        1. Just add the file and make a section titled AKS CI
        2. If you need to add information about preparing your cluster for helm chart, reference the readme in the dicom helm chart.